controllers/tweet.js

```diff
@@ -1,14 +1,17 @@
 const pool = require("../db");
 const { getCommentsByTweet, deleteComment } = require("./comment");
 
+async function asyncForEach(res) {
```

Comment:
this is not an actual async for each, but you only moved the code here. replace with the asyncForEach and pass the callback function